lib/maintainers.nix

```diff
@@ -152,7 +152,6 @@
   joachifm = "Joachim Fasting <joachifm@fastmail.fm>";
   joamaki = "Jussi Maki <joamaki@gmail.com>";
   joelmo = "Joel Moberg <joel.moberg@gmail.com>";
-  joelteon = "Joel Taylor <me@joelt.io>";
```

Comment:
I think you just broke hydra's evaluation of a bunch of other packages that refer to `joelteon`. Might want to fix those, too!